search/src/components/Contexts/DatasetAndUserContext.tsx

```diff
@@ -157,7 +169,7 @@ export const DatasetAndUserContextWrapper = (
       const params = new URLSearchParams(window.location.search);
 
       params.set("organization", organization.id);
-      window.history.replaceState(
+      window.history.pushState(
```

Comment:
is there a reason to use push state instead of replace state?

Reply:
Replace does not add a history record. It just replaces the current on. 

Push adds one which lets the back button work correctly